lib/src/tests/snapshots/typstfmt_lib__tests__math__mathblock7__snapshot.snap

```diff
@@ -0,0 +1,6 @@
+---
+source: lib/src/tests/math.rs
+description: "INPUT\n===\n\"$\\nmat(\\n    1,2,3,4;\\n5,2,2,3\\n)\\n$\"\n===\n$\nmat(\n    1,2,3,4;\n5,2,2,3\n)\n$\n===\nFORMATTED\n===\n$\n  mat(1, 2, 3, 4;5, 2, 2, 3)\n$"
+expression: formatted
+---
+"$\n  mat(1, 2, 3, 4;5, 2, 2, 3)\n$"
```

Comment:
not sure if this is in scope for this PR but we might want a space to be present after `;` in the formatted result, what's your opinion?

If comma and semicolon are the only things that can appear in math functions as argument separator then maybe, If so it would be good to add a little //TODO in your PR so that we may remember it later!

Reply:
I would like a new line there that makes it similar to the matrix.

Reply:
I don't know whether matrix is a special syntax since I cannot find a way to define a function similar to the matrix. If only matrix behaves like this I would even want to align the commas.

Reply:
We should be careful about giving special meaning to `;` and breaking lines, we don't want to end up with
```typ
$func_call(a;
b;
c;
)$

Reply:
yeah that makes thing much complicate, and I don't it to delay this pr.